packages/jellyfish-api-core/src/category/governance.ts

```diff
@@ -146,6 +146,8 @@ export class Governance {
    * @param {number} [options.pagination.start=0]
    * @param {boolean} [options.pagination.including_start=true] defaults to false if options.pagination.start is set, true otherwise
    * @param {number} [options.pagination.limit=100] to limit number of records
+   * @param {boolean} [options.aggregate=false] false: return raw vote data, true: return total votes by type
```

Comment:
can we add some tests when this is false and true? also curious if this flag would work well with other param

Reply:
Working on tests ATM. 

The flags do work together, when valid is true then aggregate information for only valid votes is shown, otherwise aggregate information for invalid votes is returned.